create_base_image/create_base_image_gce.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+
+sudo apt-get update
+
+# Stuff we need to get build support
+
+sudo apt install -y debhelper ubuntu-dev-tools equivs cloud-utils \
+  "${extra_packages[@]}"
```

Comment:
You can delete this if it isn't used.

Reply:
I used `cloud-utils` to resize the build disk, I think the others are for building debian packages and dependencies. I'll remove the `"${extra_packages[@]}"`

Reply:
yes, I meant only the extra_packages, I thought that you could select an arbitrary chunk of text for the comment like in gerrit, but github apparently only does lines